dev/TestMailer.php

```diff
@@ -4,14 +4,14 @@
  * @subpackage email
  */
 class TestMailer extends Mailer {
-	protected $emailsSent = array();
+	protected static $emailsSent = array();
```

Comment:
Fantastic fix - I had this problem the other day and just gave up testing emails, I think there's even an issue open about this